### PR TITLE
feat: add events, seperate api/node, add status codes

### PR DESCRIPTION
**Pull request by zachferland**

### CODE REVIEW

File: src/analytics.js [hunk 1]
```diff
@@ -14,58 +14,127 @@ class Analytics {
...
+  //   this._track(data)
+  // }
 
-  trackOpenDB (address, duration) {
+  trackPinDB (odbAddress) {
     let data = {}
-    data.event = 'open_db'
-    data.properties = { address: address, duration: duration }
+    data.event = 'pin_db'
+    data.properties = { odb_address: odbAddress }
```
Comment (oed) [inline src/analytics.js]: We probably don't want to track the odbAddress but rather the hash of it?

**Comment by oed** [comment]
Not sure if this is ready for review but looks good for now.

File: src/analytics.js (hunk 1 above)
Comment (zachferland) [reply src/analytics.js]: going to keep same as before for this PR, but yeah will likely switch to some form of that, but will wait to we can look at holistically across services

**Comment by zachferland** [comment]
Will add  this PR to release, and do release this evening

When releasing also have to add env var to enable metrics

**Pull request merged**